roles/splunk_common/tasks/install_splunk.yml

```diff
@@ -1,5 +1,20 @@
 ---
-- name: Install Splunk
+- name: Download splunk package
```

Comment:
remote_src variable in the task below controls if the src is remote and needs to be fetched so I think this task might be doing a repetitive job.

Reply:
Yes, I made this task for windows (check the "when:") because it doesn't use remote_src.